lib/chef/knife/exec.rb

```diff
@@ -26,7 +27,7 @@ class Chef::Knife::Exec < Chef::Knife
   option :exec,
     short: "-E CODE",
     long: "--exec CODE",
-    description: "A string of Chef code to execute."
+    description: "A string of #{Chef::Dist::PRODUCT} code to execute."
```

Comment:
I think would be fine to get us moving, so I'm not commenting asking for a change.

But ...

Usages like this demonstrate the muddiness of the water when it comes to things we all call "chef." The "string of code" to be executed here is the same whether it is being executed by the Infra Chef® chef-client® or by CINC's cinc-client or by some other build of this code base. Is this code maybe "recipe code" because the input to this option would look an awful lot like code someone would put in a recipe?

Names are hard.

Reply:
perhaps the DSL needs a name, distinct from the product or any wordmarks?

Reply:
I'd leave this one as-is for now.